README.md

```diff
@@ -5,6 +5,16 @@ SPDX-License-Identifier: GPL-2.0-or-later
 
 # userstorage
 
+[![](https://img.shields.io/pypi/v/userstorage)](https://pypi.org/project/userstorage/) 
+
+[![](https://img.shields.io/pypi/pyversions/userstorage)](https://pypi.org/project/userstorage/) 
+
+[![](https://img.shields.io/pypi/dm/userstorage)](https://pypi.org/project/userstorage/) 
+
+[![](https://github.com/nirs/userstorage/actions/workflows/ci.yml/badge.svg)](https://github.com/nirs/userstorage/actions/workflows/ci.yml) 
+
+[![](https://img.shields.io/github/license/nirs/userstorage)](https://pypi.org/project/userstorage/)
```

Comment:
Looks good except the extra blank line between the badges, which renders one badge per
line, instead of all badges on the same line:
https://github.com/nirs/userstorage/blob/39623e381c3e8ae5cf7abe766498fd379ac92364/README.md